contingency/contingency-api/src/test/resources/injectionCriterionContingencyListNoCountry.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type" : "injectionCriterion",
+  "version" : "1.0",
+  "name" : "list2",
+  "identifiableType" : "GENERATOR",
+  "countryCriterion" : null,
```

Comment:
I'm confused: why is null here and not in `injectionCriterionContingencyListEmptyCountryList`?

Reply:
I changed the json test file name to make it more explicit: there's one test for an injectionCriterion that has no country criterion and one test for an injectionCriterion with a country criterion which its country list is an empty list.